src/modding/ModManager.cs

```diff
@@ -857,6 +1308,251 @@ private void OpenInfoUrlPressed()
         }
     }
 
+    private void OnDependencyPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (selectedMod is null)
+        {
+            return;
+        }
+
+        otherModInfoDialog.WindowTitle = TranslationServer.Translate("MOD_DEPENDENCIES");
+        var infoText = string.Empty;
+
+        var currentModDependencies = selectedMod.Info.Dependencies;
+        if (currentModDependencies != null)
+        {
+            foreach (string currentDependency in currentModDependencies)
+            {
+                if (!string.IsNullOrWhiteSpace(currentDependency))
+                {
+                    infoText += "* " + currentDependency + "\n";
+                }
+            }
+        }
+        else
+        {
+            infoText += TranslationServer.Translate("NO_MOD_DEPENDENCIES");
+        }
+
+        otherModInfoDialog.DialogText = infoText;
+        otherModInfoDialog.PopupCenteredShrink();
+    }
+
+    private void OnRequiredModsPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (selectedMod is null)
+        {
+            return;
+        }
+
+        otherModInfoDialog.WindowTitle = TranslationServer.Translate("MOD_REQUIRED_MODS");
+        var infoText = string.Empty;
+
+        var currentModRequiredMods = selectedMod.Info.RequiredMods;
+        if (currentModRequiredMods != null)
+        {
+            foreach (string currentRequiredMod in currentModRequiredMods)
+            {
+                if (!string.IsNullOrWhiteSpace(currentRequiredMod))
+                {
+                    infoText += "* " + currentRequiredMod + "\n";
```

Comment:
This should use a translated string with placeholders to allow customizing. Also the default should have "-" instead of a star as that's used elsewhere in the game (and very commonly) for lists.